script.js

```diff
@@ -0,0 +1,130 @@
+            </ul>
+  `;
+
+  // Show the popup
+  popupContainer.style.display = 'block';
+}
+
+// Add event listeners for each popup button
+const popupButton1 = document.getElementById('popupButton1');
+popupButton1.addEventListener('click', () => {
+  openPopup(
+    'Tonic',
+    ' ',
+    './images/Snapshoot Portfolio.png',
+    './images/SnapshootPortfolio.svg',
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    'Button 1 for Popup 1',
+    'Button 2 for Popup 1',
+  );
+});
+
+const popupButton2 = document.getElementById('popupButton2');
+popupButton2.addEventListener('click', () => {
+  openPopup(
+    'Multi-Post Stories',
+    ' ',
+    './images/Snapshoot 2.png',
+    './images/SnapshootPortfolio.svg',
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    'Button 1 for Popup 2',
+    'Button 2 for Popup 2',
+  );
+});
+
+const popupButton3 = document.getElementById('popupButton3');
+popupButton3.addEventListener('click', () => {
+  openPopup(
+    'TONIC',
+    ' ',
+    './images/Snapshoot 3.png',
+    './images/SnapshootPortfolio.svg',
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    'Button 1 for Popup 3',
+    'Button 2 for Popup 3',
+  );
+});
+
+const popupButton4 = document.getElementById('popupButton4');
+popupButton4.addEventListener('click', () => {
+  openPopup(
+    'Multi-Post Stories',
+    ' ',
+    './images/Snapshoot Portfolio 4.png',
+    './images/SnapshootPortfolio.svg',
+    'Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industrys standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book. It has survived not only five centuries, but also the leap into electronic typesetting, remaining essent',
+    'Button 1 for Popup 4',
+    'Button 2 for Popup 4',
+  );
+});
+
+closeButton.addEventListener('click', () => {
+  popupContainer.style.display = 'none';
+});
+
+button1.addEventListener('click', () => {
+
+});
+
+button2.addEventListener('click', () => {
+
+});
```

Comment:
- [ ] Great job on implementing the hamburger menu as specified in the project requirement! 👏 However, there is a minor UI issue 🐞 that we need to address and fix to help create a good user experience. 
   - [ ] Kindly consider removing these `event` listeners from your codebase as they are not serving any purpose. Having unused code adds unnecessary complexity and potential performance overhead. Removing these unused event listeners will help improve code clarity and maintainability.

Reply:
Thank you so much!!

I removed:

button1.addEventListener('click', () => {

});

button2.addEventListener('click', () => {

});

Code is working.